sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/implementation/ManagementChannel.java

```diff
@@ -282,6 +317,87 @@
         });
     }
 
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public Flux<Long> schedule(final List<ServiceBusMessage> messages, Instant scheduledEnqueueTime, int maxLinkSize,
```

Comment:
Can we condense this? It is a copy of all the logic from the single send. same with cancel.

Reply:
merged two into one